unimport/__main__.py

```diff
@@ -63,6 +71,19 @@ def show(
         )
 
 
+def get_exclude_list_from_gitignore() -> List[str]:
+    """Converts .gitignore patterns to regex and return this exclude regex
+    list."""
+    path: Path = Path(".gitignore")
```

Comment:
Why did you need to explicitly type this?

Reply:
Unfortunately, it's a useless type annotation!

Reply:
Anyway, I'm gonna remove it with my next PR, so thx!